packages/analytics-browser/src/config/joined-config.ts

```diff
@@ -30,13 +30,11 @@ export class BrowserJoinedConfigGenerator {
         this.remoteConfigFetch &&
         (await this.remoteConfigFetch.getRemoteConfig('analyticsSDK', 'browserSDK', this.config.sessionId));
       this.config.loggerProvider.debug('Remote configuration:', JSON.stringify(remoteConfig, null, 2));
-      if (remoteConfig && remoteConfig.defaultTracking) {
-        this.config.defaultTracking = remoteConfig.defaultTracking;
-      }
-      if (remoteConfig && remoteConfig.autocapture) {
+      if (remoteConfig && 'autocapture' in remoteConfig) {
         this.config.autocapture = remoteConfig.autocapture;
+        this.config.defaultTracking = this.config.autocapture;
```

Comment:
What will happen if autocapture has userInteraction value but it's not matching the type of defaultTracking?

Reply:
I will also pass userInteraction to config.defaultTracking because `DefaultTrackingOptions` is an interface not a type. The test `should disable userInteractions if remote config sets it to false` should cover this case and make sure there is not run time error.